.github/workflows/tests.yml

```diff
@@ -36,6 +36,12 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
+      - name: Install libpq 14
```

Comment:
What I would do is to add a "libpq" entry to the grid above, and mix a few 14 and a few null to the existing steps. We can use conditional instructions and decide to install libpq 14 only on that value. This would allow to verify there is no regression when an older libpq is used, and that we can use pipeline mode also with older server versions.

Please rebase your MR on current master, because yesterday I've changed the grid massively, after introducing PG14 and Py3.10 support.

Reply:
Good idea, done.

Also took the liberty to use "postgres:14" docker image, which is now available.